jQuery/.stylelintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "stylelint-config-standard",
+    "rules": {
+        "selector-pseudo-element-no-unknown": [ true, {"ignorePseudoElements": ["ng-deep"]}],
```

Comment:
It looks like we don't need the ```ng-deep``` rule for jQuery :)

If we will create a separate ```eslintrc``` for each platform - let's remove it.